test/parallel/test-net-localerror.js

```diff
@@ -1,19 +1,19 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
 
 connect({
   host: 'localhost',
   port: common.PORT,
   localPort: 'foobar',
-}, 'localPort should be a number: foobar');
+}, /^"localPort" option should be a number: foobar$/);
```

Comment:
@Intregrisist you probably also have to add the `TypeError` part or it will fail:

```js
 /^TypeError: "localPort" option should be a number: foobar$/
```